cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -72,8 +72,8 @@ If the index is not explicitly named, it will get an auto-generated name.
 The index name must be unique among both indexes and constraints.
 
 The command is optionally idempotent, with the default behavior to throw an error if you attempt to create the same index twice.
-With `IF NOT EXISTS`, no error is thrown and nothing happens should an index with the same name, schema or both already exist.
-It may still throw an error should a conflicting constraint exist.
+With `IF NOT EXISTS`, no error is thrown and nothing happens if an index with the same name, schema or both already exists.
+It may still throw an error if conflicting data or constraints exist.
```

Comment:
```suggestion
It may still throw an error if conflicting data or constraints exists.
```
?

Reply:
constraint_s_ = plural -> exist, without the s.